code/modules/paperwork/desk_bell.dm

```diff
@@ -0,0 +1,83 @@
+// A receptionist's bell
+
+/obj/structure/desk_bell
+	name = "desk bell"
+	desc = "The cornerstone of any customer service job. You feel an unending urge to ring it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "desk_bell"
+	layer = OBJ_LAYER
+	anchored = FALSE
+	pass_flags = PASSTABLE // Able to place on tables
+	max_integrity = 5000 // To make attacking it not instantly break it
```

Comment:
There's probably a better way to do this than having 5000 integrity

Reply:
It could be flagged indestructible but that would make it survive bombs

Reply:
agreed. this is kinda silly. I feel like instead of incrementing the times rang on attack, we should just check if... the max_integrity is less than half? and then set the max_integrity to something sane.